pmlc/dialect/pxa/transforms/resize_tmps.cc

```diff
@@ -38,6 +38,14 @@ struct ResizeTmpsPass : public ResizeTmpsBase<ResizeTmpsPass> {
     Block *opBlock = op.getOperation()->getBlock();
     IVLOG(2, "Considering: " << debugString(*op.getOperation()));
 
+    for (auto &use : getIndirectUses(op)) {
+      IVLOG(2, "Found indirect use: " << debugString(*use.getOwner()));
```

Comment:
Log level 2 is probably a too high for what is mostly debugging output unlikely to be generally useful, maybe log level 4 or just remove for the commit. The other logs are fine as they are describe why the pass did what it did.

Reply:
Deleted.  I agree it is not needed.